src/gba/hardware.c

```diff
@@ -293,7 +293,13 @@ void _rtcUpdateClock(struct GBACartridgeHardware* hw) {
 		t = time(0);
 	}
 	struct tm date;
+#ifdef __CELLOS_LV2__
+	struct tm* tmpdate;
```

Comment:
Why is this variable even needed? You can just immediately deference and assign the result.

Reply:
Absolutly true, my fault... seen `localtime_r(&t, &date);` i copied and pasted my previous piece of code without to see` struct tm date;` :P